1-hello-world/app.js

```diff
@@ -25,13 +25,16 @@ app.get('/', function(req, res) {
 });
 // [END hello_world]
 
-
-// [START server]
-// Start the server
-var server = app.listen(process.env.PORT || 8080, function () {
-  var host = server.address().address;
-  var port = server.address().port;
-
-  console.log('App listening at http://%s:%s', host, port);
-});
-// [END server]
+if (module === require.main) {
```

Comment:
Is there any way to accomplish what you want without adding this extra code to the samples?